composer.json

```diff
@@ -16,7 +16,7 @@
         "php": "^7.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.5",
+        "phpunit/phpunit": "^6.5|^7.0",
```

Comment:
I only like to support one major version of PHPUnit at a time, but it looks like we can bump this to `^7.0` now :+1:

Reply:
Oh yes. I will change into using version ```^6.5``` now.